build/docker/README.md

```diff
@@ -46,14 +46,13 @@ Docker Hub and push the resulting tags.
 
 ## Supported Containers ##
 
-The Travis CI (continuous integration) builds use the Ubuntu Bionic
-(18.04 LTS) and Xenial (16.04 LTS) images to maximize language level
+The Travis CI (continuous integration) builds use the Ubuntu Jammy
+(22.04 LTS) and Focal (20.04 LTS) images to maximize language level
 coverage.
 
 ### Ubuntu ###
 
 * focal (stable, current)
-* bionic (previous stable)
 * jammy (next stable, WIP)
```

Comment:
now jammy is already the current stable and focal is already the previous stable, right? https://ubuntu.com/about/release-cycle